model/src/guild/integration_expire_behavior.rs

```diff
@@ -1,17 +1,40 @@
-use serde_repr::{Deserialize_repr, Serialize_repr};
+use serde::{Deserialize, Serialize};
 
 /// Behavior to perform when the user's integration expires.
 #[derive(
-    Clone, Copy, Debug, Deserialize_repr, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize_repr,
+    Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, PartialOrd, Serialize, Ord,
 )]
-#[repr(u8)]
+#[serde(from = "u8", into = "u8")]
 pub enum IntegrationExpireBehavior {
     /// Remove the role when the integration expires.
-    RemoveRole = 0,
+    RemoveRole,
     /// Kick the user when the integration expires.
-    Kick = 1,
+    Kick,
+    /// Unknown what it'll do when the integration expires
```

Comment:
I think just having the aforementioned standard unknown documentation will make it clear that the behavior of the integration expiry is unknown.